controllers/runnerpool_controller_test.go

```diff
@@ -295,6 +295,7 @@ var _ = Describe("RunnerPool reconciler", func() {
 			"Resources": MatchAllFields(Fields{
 				"Limits":   BeEmpty(),
 				"Requests": BeEmpty(),
+				"Claims":   BeEmpty(),
```

Comment:
From k8s.io/api v0.26.0, the `ResourceRequirements` has the `Claims` field.

https://github.com/kubernetes/api/blob/v0.26.4/core/v1/types.go#L2317-L2329